tests/integrations/mcs/tso/proxy_test.go

```diff
@@ -75,17 +77,12 @@ func (s *tsoProxyTestSuite) SetupSuite() {
 		Count:  1,
 	}
 
-	// Create some TSO client streams with the same context.
-	s.grpcClientConns, s.streams, s.cancelFuncs = createTSOStreams(re, s.ctx, s.backendEndpoints, 100, true)
-	// Create some TSO client streams with the different context.
-	grpcClientConns, streams, cancelFuncs := createTSOStreams(re, s.ctx, s.backendEndpoints, 100, false)
-	s.grpcClientConns = append(s.grpcClientConns, grpcClientConns...)
-	s.streams = append(s.streams, streams...)
-	s.cancelFuncs = append(s.cancelFuncs, cancelFuncs...)
+	// Create some TSO client streams with different context.
+	s.streams, s.cleanupFuncs = createTSOStreams(re, s.ctx, s.backendEndpoints, 200)
```

Comment:
different context.?

Reply:
Yes, inside of createTSOStreams, it will use different context to create "stream, err := grpcPDClient.Tso(cctx)" PD_TsoClient. The result is that the stream.Context() on the server side is different.